control-plane/plugin/src/resources/utils.rs

```diff
@@ -22,7 +22,7 @@ lazy_static! {
         "THIN-PROVISIONED",
         "ALLOCATED"
     ];
-    pub static ref SNAPSHOT_HEADERS: Row = row!["ID", "TIMESTAMP", "SIZE", "SOURCE-VOL"];
+    pub static ref SNAPSHOT_HEADERS: Row = row!["ID", "TIMESTAMP", "ALLOCATED-SIZE", "SOURCE-VOL"];
```

Comment:
Should we add SOURCE-SIZE and TOTAL-SIZE as well?

Reply:
Yeah we can. Adding it.